playgrounds/feature-service-maven-sb-3/docker/Dockerfile

```
@@ -1,9 +1,52 @@
+# Step 1: Record class loading profile
+RUN java \
+    -XX:+UseCompactObjectHeaders \
+    -XX:AOTMode=record \
+    -XX:AOTConfiguration=application.aotconf \
+    -Daxelix.sbs.auth.jwt.algorithm=HMAC256 \
+    -Daxelix.sbs.auth.jwt.signing-key=8DrZJSOJ8vkbxdjUB3sSsyeiG4Xidf1sDNmJq1Slkkn \
+    -Dspring.context.exit=onRefresh \
+    -jar /application/application.jar
+
+# Step 2: Create AOT cache from the recorded profile
+RUN java \
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)